functions.js

```diff
@@ -34,10 +37,10 @@ function createResponseObject(resolve, reject) {
     error: function(error) {
       reject(new Parse.Error(Parse.Error.SCRIPT_FAILED, error));
     }
-  }
+  };
 }
 
-router.route('POST', '/functions/:functionName', handleCloudFunction);
+router.route('POST', '/functions/:functionName+', handleCloudFunction);
```

Comment:
It requires latest express and path-to-regexp version, otherwise it will fallback to initial behaviour.